### PR TITLE
add banner for upcoming 19.0.0 release

### PR DESCRIPTION
**Pull request by Trott**
This can't land until the 19.0.0 release announcement is posted.

### CODE REVIEW

**Comment by Trott** [comment]
Added the `on hold` label because this shouldn't land until:

1. The 19.x release is out.
2. The 19.x release blog post is in place.
3. It is after 9am Pacific Time on October 18, 2022.

If all three of those are true, feel free to remove the `on hold` label and land this.

**Comment by Trott** [comment]
@nodejs/releasers

File: locale/en/site.json
Comment (nschonni) [inline locale/en/site.json]: Not sure if the absolute vs FQDN might have an issue downstream on `.dev`. That might have been worked around last time though

Comment (ovflowd) [reply locale/en/site.json]: FQDN is needed, otherwise, .dev will break 😅

Comment (Trott) [reply locale/en/site.json]:
```suggestion
      "html": "<img alt='Node.js 19 is now live!' src='https://nodejs.org/static/images/NodeJS19_Banner.png'>",
```

File: locale/en/site.json
Comment (BethGriggs) [inline locale/en/site.json]: I think in the past we've pointed to the announcement post rather than the changelog-style one.

I don't know the slug of that post yet - but it was https://nodejs.org/de/blog/announcements/v18-release-announce/ for 18.

Comment (Trott) [reply locale/en/site.json]: OK, I'll switch it to that format and if that doesn't end up being the right slug, we can update this again. 

```suggestion
      "link": "https://nodejs.org/en/blog/announcements/v19-release-announce/"
```

**Review by RafaelGSS — APPROVED** [review]

**Review by ruyadorno — APPROVED** [review]

**Review by BethGriggs — APPROVED** [review]

**Comment by Trott** [comment]
It looks like this landed before 9am Pacific, which means the banner won't show, I don't think until something else lands.

**Comment by Trott** [comment]
I'm going to force-push a no-op in the hope that will cause the banner to be deployed.

**Comment by richardlau** [comment]
The website was rebuilt and I can see a cdn purge is queued. The queue is processed every five minutes so the website should update around 10 mins past the hour.

**Pull request merged**
